OpenRA.Game/Settings.cs

```diff
@@ -72,7 +72,7 @@ public class ServerSettings
 		[Desc("Disallow AI bots.")]
 		public bool LockBots = false;
 
-		public string TimestampFormat = "HH:mm";
+		public string TimestampFormat = "dd/MM - HH:mm:ss";
```

Comment:
Could we use [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) (`YYYY-MM-ddTHH:mm:ssZ`, but there's probably a short code for it) instead?